packages/myst-templates/src/frontmatter.ts

```diff
@@ -65,24 +69,20 @@ function addIndicesToAuthors(
   });
 }
 
-function affiliationsFromAuthors(authors: Author[]): ValueAndIndex[] {
-  const allAffiliations = authors.map((auth) => auth.affiliations || []).flat();
-  return [...new Set(allAffiliations)].map((value, index) => {
-    return { value, ...indexAndLetter(index) };
-  });
-}
-
-function collaborationsFromAuthors(authors: Author[]): ValueAndIndex[] {
-  const allCollaborations = authors.map((auth) => auth.collaborations || []).flat();
-  return [...new Set(allCollaborations)].map((value, index) => {
-    return { value, ...indexAndLetter(index) };
-  });
-}
-
 export function extendFrontmatter(frontmatter: PageFrontmatter): RendererDoc {
   const datetime = frontmatter.date ? new Date(frontmatter.date) : new Date();
-  const affiliations = affiliationsFromAuthors(frontmatter.authors || []);
-  const collaborations = collaborationsFromAuthors(frontmatter.authors || []);
+  const affiliations =
+    frontmatter.affiliations
+      ?.filter((aff) => aff.id && !aff.collaboration)
+      .map((aff, index) => {
+        return { value: aff, ...indexAndLetter(index) };
```

Comment:
These changes will need to be consumed in the tex templates themselves; now, affiliations are objects instead of strings.

Reply:
There is also going to be a bit around @dressedfez's template that consumes the collaboration.

I think that they should be straight forward, however, we don't really have a way to version the templates which means this is a breaking change in that respect. Any better thoughts on how to handle it?

Reply:
I would be willing to introduce a versioning on my side. @rowanc1 which changes are necessary in the template?